### ISSUE TITLE
Weird "5Hz2" ODR constant in several drivers

### ISSUE THREAD

**Issue by arbrauns** [STMicroelectronics/STMems_Standard_C_drivers#162]
### Device part numbers

```
$ rg --files-with-matches odr_5hz2 | cut -d/ -f1 | sort -u
ais328dq_STdC
ais3624dq_STdC
iis328dq_STdC
```

### Type of bug

driver

### Describe the bug

Several drivers have a `5Hz2` output data rate (ODR) constant with a value that represents 2Hz according to the datasheet. Presumably this is a typo that has been blindly copy-pasted ever since.

**Comment by avisconti** [comment]
Of course it is 2Hz. Thanks for noticing it. I'll apply changes to all impacted drivers

**Comment by avisconti** [comment]
Closed by e5715ca2dbbbc4d68b844364a693364f8849404c
